backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitChannelPool.java

```diff
@@ -60,6 +60,12 @@ public Channel create() throws Exception {
         public PooledObject<Channel> wrap(Channel obj) {
             return new DefaultPooledObject<>(obj);
         }
+
+        @Override
+        public void destroyObject(PooledObject<Channel> pooledObject) throws Exception {
```

Comment:
Is this method used?

Reply:
inside the `PooledObject.close()` it calls `PooledObjectFactory.destroyObject()`. The close is actually happens inside `PooledObjectFactory`